src/test/java/net/snowflake/client/jdbc/ResultSetAsyncIT.java

```diff
@@ -270,6 +270,7 @@ public void testGetMethods() throws Throwable {
             prepStatement.setTimestamp(14, ts);
             prepStatement.execute();
 
+            statement.execute("alter session set JDBC_FORMAT_DATE_WITH_TIMEZONE=FALSE");
```

Comment:
why do we need to update tests in this class?

the same question for other integration tests

Reply:
See comment above